homeassistant/components/alarm_control_panel/demo.py

```diff
@@ -0,0 +1,38 @@
+"""
+homeassistant.components.alarm_control_panel.demo
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Demo platform that has two fake alarm control panels.
+"""
+import homeassistant.components.alarm_control_panel.manual as Alarm
+from homeassistant.const import (STATE_ALARM_DISARMED,
+                                 STATE_ALARM_ARMED_AWAY)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    """ Sets up the Demo alarm control panels. """
+    add_devices([
+
+        DemoAlarmControlPanel(hass, 'Front door', '1234', 2, 4,
+                              STATE_ALARM_DISARMED),
+        DemoAlarmControlPanel(hass, 'Safe', '1234', 2, 4,
+                              STATE_ALARM_ARMED_AWAY),
+        ])
+
+
+# pylint: disable=too-many-arguments
+class DemoAlarmControlPanel(Alarm.ManualAlarm):
```

Comment:
There doesn't seem to be a need for this class. You can just add `Alarm.ManualAlarm` directly.